### PR TITLE
Create tests to check the creation of objects in the Storage Handler

### LINKED ISSUE #47: Add tests to check the creation of objects in the Storage Handler
Currently, there are no tests to verify the creation of Person, Book, and Club objects in the Storage Handler calls to the database.

### PR DESCRIPTION
**Pull request by linamontes10-zz**
Adds tests to verify the object creation of Person, Book, and Club objects in `getPerson`, `getBook`, and `getClub`.
Closes #47 .

### CODE REVIEW

File: server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTest.java [hunk 1]
```diff
@@ -47,6 +50,139 @@ public void setUp() {
...
+      StorageHandler.getPerson(dbClient, "personNotInDb");
+    });
+  }
+
+  @Test
+  public void getPerson_existsWithPronouns() throws Exception {
+    StorageHandlerTestHelper.insertPersonWithPronouns("person");
+    Person actual = StorageHandler.getPerson(dbClient, "person");
+    Person expected = StorageHandlerTestHelper.createTestPersonObject(
+      "person", /**pronouns = **/true);
```
Comment (KeatonTech) [inline server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTest.java]: super minor nit: Inline comments just use `/* ... */`. The double-asterisks are for JSDoc comments

**Review by KeatonTech — APPROVED** [review]

File: server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTest.java (hunk 1 above)
Comment (linamontes10-zz) [reply server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTest.java]: Thank you for pointing this out!

File: server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java [hunk 2]
```diff
@@ -71,7 +71,7 @@ public static Person getPerson(DatabaseClient dbClient, String userId) {
                                            .setNickname(row.getString(/* nicknameIndex= */1))
                                            .setUserId(userId);
 
-      if (!row.isNull(/* index= */2) || !row.getString(/* index= */2).isEmpty()) {
+      if (!row.isNull(/* index= */2)) {
```
Comment (jimmycolema) [inline server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java:74]: super nit: for this inline comment, there is no space between the variable and the equals sign, while below you have a space between. I believe the no space is the correct format, but I would choose one for styling consistency

**Review by jimmycolema — COMMENTED** [review]
lgtm!

File: server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java (hunk 2 above)
Comment (linamontes10-zz) [reply server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java:74]: I will change to no space between the inline comments, thank you for catching this style issue!

File: server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTestHelper.java [hunk 3]
```diff
@@ -111,190 +139,135 @@ public static void insertBook(String book_id) {
...
-        .to(title)
-        .build());
-    dbClient.write(mutations);
+  public static Book createTestBookObject(String book_id, Boolean isbn_exists,
+                                            Boolean author_exists) {
+    Book.Builder bookBuilder = Book.newBuilder()
+                                   .setTitle("title")
+                                   .setBookId(book_id);
+    if (isbn_exists) { bookBuilder.setIsbn("isbn"); }
+    if (author_exists) { bookBuilder.setAuthor("author"); }
```
Comment (JosephBushagour) [inline server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTestHelper.java]: Technically Google style guide only allows concise brackets when they're empty (https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style). This should be reformatted to not use concise brackets.

**Review by JosephBushagour — APPROVED** [review]
Nice!

File: server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTestHelper.java (hunk 3 above)
Comment (linamontes10-zz) [reply server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTestHelper.java]: Got it, thanks for pointing this out. I was trying to save some space, but will change this to adhere to the style guide.

**Pull request merged**